qplist.pri

```diff
@@ -0,0 +1 @@
+LIBS += 
```

Comment:
Having to add a .pri saying I don't need any extra libs seems like a bug in qt-pods.